### PR TITLE
added a couple more

### PR DESCRIPTION
**Pull request by JonW27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # free-for-dev
 Developers and Open Source authors now have a massive amount of services offering free tiers, but it can be hard to find them all in order to make informed decisions.
 
-This is list of software (SaaS, PaaS, IaaS, etc.) and other offerings that have free tiers for developers.
+This is list of software (SaaS, PaaS, IaaS, etc.) and other offerings thatƒ have free tiers for developers.
```
Comment (ripienaar) [inline README.md]: some unintended edit here I guess?

File: README.md [hunk 2]
```diff
@@ -91,6 +91,8 @@ Table of Contents
   * https://codenvy.com/ - Share developer environments for your project. Includes configuration for the browser IDE, build, debug and run environments. Free tier with 4 GB environments and support for 90+ languages
   * http://wistia.com/ - Video hosting with viewer analytics, HD video delivery, and marketing tools to help understand your visitors, 25 videos and Wistia branded player
   * https://cnverg.com/ - Real-time shared visual workspace, whiteboard, GitHub integration. Free 5 GB, 5 spaces and 5 collaborators, no GitHub repos
+  * https://www.flowdock.com/ - Free for teams of five and students.  Unlimited flows. Unlimited history. Unlimited integrations.
```
Comment (ripienaar) [inline README.md]: we already have this one elsewhere

**Comment by ripienaar** [comment]
looks good otherwise :)

File: README.md (hunk 1 above)
Comment (JonW27) [inline README.md]: yeah unintended.

**Comment by ripienaar** [comment]
let me know when you edited out the unintended edit and I'll merge :)

**Comment by ripienaar** [comment]
closing, please re-open if you're still keen on getting these in

**Pull request closed without merging**